popup/popup.css

```diff
@@ -1,17 +1,19 @@
 html, body {
-    width: 500px;
+    max-width: 500px;
+    width: 100%;
     margin: 0;
-    padding: 0 10px 0 0;
+    padding: 10px;
     overflow-x: hidden;
     background: white;
+    box-sizing: border-box;
 }
 
 h1 {
     margin: 0;
```

Comment:
When data is reset, the popup width is too small (Firefox on Ubuntu)

![Screenshot from 2019-07-17 22-06-10](https://user-images.githubusercontent.com/1101620/61407978-475d9700-a8df-11e9-951d-d9877e65b38b.png)

I fixed it by adding here in h1:

    width: 500px;